recipes/llama-index-retrievers-bm25/recipe.yaml

```diff
@@ -0,0 +1,45 @@
+context:
+  version: 0.5.2
+
+package:
+  name: llama-index-retrievers-bm25
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/l/llama-index-retrievers-bm25/llama_index_retrievers_bm25-${{ version }}.tar.gz
+    sha256: e7cd84565dc9b523bc2263cf75c607c9e5e68ce0edc8a24d2ac6a837b1de557f
+  - url: https://raw.githubusercontent.com/run-llama/llama_index/main/LICENSE
+    sha256: 24f40b5190fdacabc24ddbb5f76364d15e4f030925220ea300d8a2dd4993c8cb
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install .
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - pip
+    - poetry
```

Comment:
should this be `poetry-core`? (looking at the upstream `pyproject.toml`)

Reply:
thanks, fixed!